src/python/pants/testutil/engine/util.py

```diff
@@ -130,6 +131,15 @@ def create_scheduler(rules, union_rules=None, validate=True, native=None):
   )
 
 
+def create_options_bootstrapper(
```

Comment:
This possibly lives in the wrong file and/or has the wrong name? I could be convinced that this really belongs in `testutil.option.util` (does not exist) or `testutil.option.fakes`. 

I have it here for discoverability, as many V2 tests already import `testutil.engine.util`, but that's not a very compelling argument and as I type this out I think I should move this to `testutil.option.util`? Thoughts?

Reply:
A `testutil.option` package location for this seems reasonable.